perun-core/src/main/java/cz/metacentrum/perun/core/impl/MembersManagerImpl.java

```diff
@@ -490,4 +490,76 @@ public MemberGroupStatus getUnifiedMemberGroupStatus(PerunSession sess, User use
 
 	}
 
+	@Override
+	public List<Member> findMembers(PerunSession sess, Vo vo, String searchString, boolean onlySponsored) throws InternalErrorException {
+		Set<Member> members = new HashSet<>();
+
+		String voIdQueryString = "";
+		if(vo != null) {
+			voIdQueryString = " members.vo_id=" + vo.getId() + " and ";
+		}
+
+		String sponsoredQueryString = "";
+		if(onlySponsored) {
+			//only sponsored members
+			sponsoredQueryString+=" members.sponsored=1 and ";
+		}
+
+		String userNameQueryString = "";
```

Comment:
NIT: You don't have to initialize it with an empty string.

Reply:
Removed.